blazor/heatmap-chart/tooltip.md

```diff
@@ -53,4 +53,451 @@ You can enable the tooltip by setting the `Enable` property to true inside the `
 
 ```
 
-![Blazor HeatMap Chart displays Tooltip](images/blazor-heatmap-chart-tooltip.png)
\ No newline at end of file
+![Blazor HeatMap Chart displays Tooltip](images/tooltip/blazor-heatmap-chart-tooltip.png)
+
+
+## Tooltip template
+
+Any custom HTML element can be rendered as a tooltip of the HeatMap using the [Template](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.HeatMap.HeatMapTooltipSettings.html#Syncfusion_Blazor_HeatMap_HeatMapTooltipSettings_Template) property of the [HeatMapTooltipSettings](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.HeatMap.HeatMapTooltipSettings.html).
+
+The following examples show how to display the tooltip template for various data bindings in the HeatMap.
```

Comment:
Retake all the images for the tooltip template. the template content is not proper.
![chrome_Mm8m4m9hMQ](https://user-images.githubusercontent.com/93366915/227184255-5d42f771-6c80-4754-9a94-a40743f7764d.png)